src/main/java/redis/embedded/RedisServer.java

```diff
@@ -7,7 +7,7 @@
 import java.util.List;
 
 public class RedisServer extends AbstractRedisInstance {
-    private static final String REDIS_READY_PATTERN = ".*The server is now ready to accept connections on port.*";
+    private static final String REDIS_READY_PATTERN = "Server initialized";
```

Comment:
Here i need the help for making the code changes generic for arm64

Reply:
Are you getting different text outputs for AMD64 and ARM64? If yes, I think it is due to redis version.

Reply:
yes